src/components/HclListItem.tsx

```
@@ -60,12 +59,10 @@ export const HclGeneralListItem: React.FunctionComponent<PropsWithChildren<{ tit
 }
 
-export const HclListItemTypeDetails: React.FunctionComponent = ({
-  children,
-}) => {
-  return (
-    <HclGeneralListItem title="Type Details" children={children} />
-  )
+export const HclListItemTypeDetails: React.FunctionComponent<
+  PropsWithChildren
+> = ({ children }) => {
+  return <HclGeneralListItem title="Type Details" children={children} />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)